modules/virt-about-instance-types.adoc

```diff
@@ -15,7 +15,7 @@ To create a new instance type, you must first create a manifest, either manually
 * A namespaced object: `VirtualMachineInstancetype`
 * A cluster-wide object: `VirtualMachineClusterInstancetype`
 
-These objects use the same `VirtualMachineInstancetypeSpec`.
+Both instance type objects use the same spec.
```

Comment:
inplace of spec should we mention VirtualMachineInstancetypeSpec?

Reply:
It doesn't really fit with our doc guidelines, but since it's not really in scope here I'll just change it back for now.